code/__DEFINES/combat.dm

```diff
@@ -158,7 +158,7 @@
 /// how long they're staggered for
 #define SHOVE_STAGGER_DURATION 35
 /// how long they're off balance for
-#define SHOVE_OFFBALANCE_DURATION 30
```

Comment:
@DeltaFire15 thoughts?
i'm against the change.

Reply:
Two seconds feels a lot like 3-4.  The window is bigger than it lets on to be.  When it's at 3 seconds it feels like 4-5.  I could change it back to 3 if we really need it, but even allowing everyone to disarm anything by getting shoved into a wall/person is still really goofy.